Database/3 reports.sql

```diff
@@ -0,0 +1,38 @@
+-- I will need the following reports:
+use ACutAboveDB
+go
+
+-- 1) upoming appointments- seperate report for each hairdresser, ordered by appointment time with more recent appointment 
+-- at the top of the list. Report should be formatted like this: Client first name client last name apt day of the week, apt datetime - service (price) phone number
+select UpcomingAppointments_Sara = CONCAT(a.ClientFirstName, ' ', a.ClientLastName, ' ', a.DayOfWeek,' ', a.AppointmentDate, ' ', a.ServiceReq, ' - ($',a.Price,') ', a.PhoneNumber )
```

Comment:
- dash should be between apt datetime and service instead of between service and price 
- there should be a comma after day of the week
- this was not in the original spec, but can you add a colon after ClientLastName in each hairdresser's report?

Reply:
Done